docs/supported-technologies.asciidoc

```diff
@@ -6,7 +6,7 @@ The Elastic APM Python Agent comes with support for the following frameworks:
 
  * <<django-support,Django>>
  * <<flask-support,Flask>>
- 
+
```

Comment:
Tornado is a framework, right? Let's add it here if so. It looks like Aiohttp isn't a framework, but we should probably add it here too. If it makes more sense, we can change the preceding sentence to remove framework. Something like: "The Elastic APM Python Agent comes with support for the following:"
```suggestion
 * <<supported-aiohttp,Aiohttp Server>>
 * <<supported-tornado,Tornado>>
```